testing/asserts.ts

```diff
@@ -545,6 +545,10 @@ export function assertObjectMatch(
   const seen = new WeakMap();
   return assertEquals(
     (function filter(a: loose, b: loose): loose {
+      // If the actual value is an array, let assertEquals do the assertion.
+      if (Array.isArray(a)) {
+        return a;
+      }
       // Prevent infinite loop with circular references with same filter
```

Comment:
```suggestion

```

Reply:
Why do you think this comment is useless?

Reply:
Comments are typically seen as a good thing, they can improve readability, offer context as to what the code is trying to do but in some case the code don’t bring any other explanation rather than the code itself.

Reply:
I don't get it. This code is complex enough that it warrants a comment that explains the purpose of it. The code reads as "make a weakmap to store things in it" but not for what purpose, the comment clarifies this.

Reply:
Comments seem fine to me, they're providing context.